lib/translator/translation.cc

```diff
@@ -68,39 +69,68 @@ BeginResponse Translation::Begin(Span<const uint8_t> scsi_cmd,
   return response;
 }
 
-ApiStatus Translation::Complete(Span<const nvme::GenericQueueEntryCpl> cpl_data,
-                                Span<uint8_t> buffer) {
+CompleteResponse Translation::Complete(
+    Span<const nvme::GenericQueueEntryCpl> cpl_data, Span<uint8_t> buffer,
```

Comment:
nit: might want to rename to `buffer_in` so it's consistent with the name in the header

Reply:
Nice catch